src/store/modules/errors.js

```diff
@@ -0,0 +1,17 @@
+const defaulState = {
+  message: '',
```

Comment:
Identifying errors with strings is considered to be a form of a weak contract with the error, a class can be used to make it clearer what the error is.

```suggestion
  error: null,
```